inputfiles/addedTypes.json

```diff
@@ -2980,5 +2980,21 @@
         "signatures": [
             "animate(keyframes: AnimationKeyFrame | AnimationKeyFrame[], options: number | AnimationOptions): Animation"
         ]
+    },
+    {
+        "kind": "method",
+        "name": "lockOrientation",
+        "interface": "Screen",
+        "signatures": [
+            "lockOrientation(orientations: string | string[]): boolean"
```

Comment:
seems that these are fixed set of strings. https://w3c.github.io/screen-orientation/#orientationtype-enum